identity_agent/benches/remote_account_benchmark.rs

```diff
@@ -0,0 +1,151 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use criterion::criterion_group;
+use criterion::criterion_main;
+use criterion::BenchmarkId;
+use criterion::Criterion;
+use identity_agent::agent::Agent;
+use identity_agent::agent::AgentBuilder;
+use identity_agent::agent::AgentId;
+use identity_agent::Multiaddr;
+
+use remote_account::IdentityCreate;
+use remote_account::RemoteAccount;
+
+async fn setup() -> (Agent, AgentId, Agent) {
+  let addr: Multiaddr = "/ip4/0.0.0.0/tcp/0".parse().unwrap();
+  let mut builder = AgentBuilder::new();
+
+  let remote_account = RemoteAccount::new().unwrap();
+  builder.attach::<IdentityCreate, _>(remote_account);
+
+  let mut receiver: Agent = builder.build().await.unwrap();
+
+  let addr = receiver.start_listening(addr).await.unwrap();
+  let receiver_agent_id = receiver.agent_id();
+
+  let mut sender: Agent = AgentBuilder::new().build().await.unwrap();
+
+  sender.add_agent_address(receiver_agent_id, addr).await.unwrap();
+
+  (receiver, receiver_agent_id, sender)
+}
+
+fn bench_create_remote_account(c: &mut Criterion) {
+  static ITERATIONS: &[usize] = &[100, 10_000, 100_000];
```

Comment:
Same comment here as in the previous benchmark file. I don't see why these iterations are necessary.